hakyll.cabal

```diff
@@ -208,13 +210,13 @@ Library
     time-locale-compat   >= 0.1      && < 0.2,
     unordered-containers >= 0.2      && < 0.3,
     vector               >= 0.11     && < 0.14,
+    wai-app-static       >= 3.1      && < 3.2,
```

Comment:
Why was this moved from under the `previewServer` flag? I build current `master` (f466be79f58b34fd94f2249d2731a31c67a2a26d) with the following command, and it works fine:

```
$ for action in build test ; do cabal $action --enable-tests --flags='-previewServer' || break ; done
```

Reply:
That's very strange, master isn't building for me:

```bash
lib/Hakyll/Core/Configuration.hs:14:1: error:
    Could not load module ‘Network.Wai.Application.Static’
    It is a member of the hidden package ‘wai-app-static-3.1.7.4’.
    Perhaps you need to add ‘wai-app-static’ to the build-depends in your .cabal file.
    Use -v (or `:set -v` in ghci) to see a list of the files searched for.
   |
14 | import qualified Network.Wai.Application.Static as Static
```

using ghc 9.4.4

Reply:
Oh, sorry, I was building your branch >_< It does fail on master for me too.